### PR TITLE
Fix cloudbeat's testing configuration - adding runtime_cfg to default config

### PR DESCRIPTION
**Pull request by kfirpeled**
Until now, without passing the proper configuration, `cloudbeat` evaluated resources against all the available benchmarks. This PR fix that behaviour to better mimic how things are working on production

### CODE REVIEW

**Review by oren-zohar — APPROVED** [review]

**Comment by oren-zohar** [comment]
@kfirpeled maybe we should also edit [cloudbeat-pytest.yml](https://github.com/elastic/cloudbeat/blob/main/tests/deploy/cloudbeat-pytest.yml) and [cloudbeat-eks-pytest.yaml](https://github.com/elastic/cloudbeat/blob/main/tests/deploy/cloudbeat-eks-pytest.yaml). Not sure why we don't use the deployment ymls you edited in our functional tests infra, @gurevichdmitry @DaveSys911 maybe you can help here

**Review by oren-zohar — CHANGES_REQUESTED** [review]
@kfirpeled maybe we should also edit [cloudbeat-pytest.yml](https://github.com/elastic/cloudbeat/blob/main/tests/deploy/cloudbeat-pytest.yml) and [cloudbeat-eks-pytest.yaml](https://github.com/elastic/cloudbeat/blob/main/tests/deploy/cloudbeat-eks-pytest.yaml). Not sure why we don't use the deployment ymls you edited in our functional tests infra, @gurevichdmitry @DaveSys911 maybe you can help here

**Comment by gurevichdmitry** [comment]
> @kfirpeled maybe we should also edit [cloudbeat-pytest.yml](https://github.com/elastic/cloudbeat/blob/main/tests/deploy/cloudbeat-pytest.yml) and [cloudbeat-eks-pytest.yaml](https://github.com/elastic/cloudbeat/blob/main/tests/deploy/cloudbeat-eks-pytest.yaml). Not sure why we don't use the deployment ymls you edited in our functional tests infra, @gurevichdmitry @DaveSys911 maybe you can help here

@oren-zohar, @kfirpeled , you are right, **cloudbeat-pytest.yml** and **cloudbeat-eks-pytest.yaml** should be updated appropriately.

**Comment by kfirpeled** [comment]
I hope I've fixed everything, I've tested the template files using https://gotemplate.io/

@gurevichdmitry I've noticed that the period is set to 5s when testing eks. Are we sure that's ok? I remember it caused issues when doing the functional tests. Checkout: `tests/deploy/cloudbeat-eks-pytest.yaml:180`

**Comment by gurevichdmitry** [comment]
> @gurevichdmitry I've noticed that the period is set to 5s when testing eks. Are we sure that's ok? I remember it caused issues when doing the functional tests. Checkout: `tests/deploy/cloudbeat-eks-pytest.yaml:180`

The period already changed in EKS functional tests PR.

**Review by gurevichdmitry — APPROVED** [review]
@kfirpeled, tests/deploy folder approved.

**Review by oren-zohar — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -160,15 +160,26 @@ Cloudbeat is only supported on managed elastic-agents. It means, that in order t
...
+```sh
+mage config
+```
+
+In order to control the policy type you can pass the following environment variable
+```sh
+POLICY_TYPE=cloudbeat/cis_eks mage config
+```
 
+The default `POLICY_TYPE` is set to `cloudbeat/cis_k8s` on `_meta/config/cloudbeat.common.yml.tmpl`
```
Comment (oren-zohar) [inline README.md:182]:
```suggestion
The default `POLICY_TYPE` is set to `cloudbeat/cis_k8s` on [`cloudbeat.common.yml.tmpl`](https://github.com/elastic/cloudbeat/blob/main/_meta/config/cloudbeat.common.yml.tmpl)
```

**Review by oren-zohar — APPROVED** [review]

**Pull request merged**